packages/workbox/index.js

```diff
@@ -57,24 +57,28 @@ function getOptions (moduleOptions) {
     modifyUrlPrefix: {
       '': fixUrl(publicPath)
     },
+    offline: true,
     _runtimeCaching: [
       // Cache all _nuxt resources at runtime
       // They are hashed by webpack so are safe to loaded by cacheFirst handler
       {
         urlPattern: fixUrl(publicPath + '/.*'),
         handler: 'cacheFirst'
-      },
-      // Cache other routes if offline
-      {
-        urlPattern: fixUrl(routerBase + '/.*'),
-        handler: 'networkFirst'
       }
     ],
     runtimeCaching: []
   }
 
   const options = defaultsDeep({}, this.options.workbox, moduleOptions, defaults)
 
+  // Optionally cache other routes for offline
+  if (options.offline) {
+    defaults._runtimeCaching.push({
```

Comment:
@pi0 Head up! I just noticed this today. This should be `options._runtimeCaching.push()` (not `defaults._runtimeCaching.push()`). 🙈

Reply:
Thanks for notifying me. Will fix it :)